src/auth-service/utils/multitenancy.js

```diff
@@ -1,29 +1,27 @@
 const mongodb = require("../config/dbConnection");
 const constants = require("../config/constants");
```

Comment:
some of these changes are not expected in this PR, please revisit the way you resolved the merge conflicts. @123MwanjeMike